src/game.cpp

```diff
@@ -6216,139 +6161,6 @@ static bool writeResearchFile(char *pFileName)
 	return true;
 }
 
-
-// -----------------------------------------------------------------------------------------
-// load up saved message file
-bool loadSaveMessage(const char *pFileName, SWORD levelType)
```

Comment:
Same thing here as with `loadMissionExtras`. Moved to `levels.cpp` as that is the only place calling this, and changing the level type param introduced a circular dependency.